### PR TITLE
Update to Ukrainian Translation

### PR DESCRIPTION
**Pull request by ZaHaDumGoth**
Artifacts description text.

### CODE REVIEW

**Review by ihhub — APPROVED** [review]

File: files/lang/uk.po [hunk 1]
```diff
@@ -9175,26 +9183,30 @@ msgstr ""
...
 msgid ""
 "Ridding the countryside of the hideous Minotaur who made a sport of eating "
 "noblemen's Knights, you are honored with the Medal of Distinction."
 msgstr ""
+"Позбавивши сільську місцевість від огидного Мінотавра, який розважався тим, "
+"що поїдав шляхетних лицарів, ви отримаєте медаль \"За відзнаку\"."
 
 msgid "Fizbin of Misfortune"
-msgstr ""
+msgstr "Брошка нещастя"
```
[the remark this answers is not in the document]
Comment (ZaHaDumGoth) [reply files/lang/uk.po:9196]: I didn't find translation to this strange word. This is not a pendant on the neck and not a coin to keep it in pocket. Thats why I chose word "Брошка". It seems to me the most suitable - some thing that can be carried on cloth as jewelery.
So, should I better make transliteration instead?

[the remark this answers is not in the document]
Comment (ZaHaDumGoth) [reply files/lang/uk.po:9196]: I assume no one understand that or say "Fizbin-what the hell is that?" :-D
We (I and Asturiel) will disscus this today, but I predict its going to be "брошка" (small item which can be fixed on clothing).
Please hold on for few hours.

[the remark this answers is not in the document]
Comment (ZaHaDumGoth) [reply files/lang/uk.po]: We decided "медаль" according to description of the artifact. In Future probably we gonna get back to this word.

**Comment by ihhub** [comment]
@ZaHaDumGoth , thank you so much for this update!

**Pull request merged**